app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -174,8 +174,10 @@ public function panel(Panel $panel): Panel
                 FilamentFullCalendarPlugin::make(),
             ])
             ->userMenuItems([
-                'profile' => MenuItem::make()
-                    ->url(fn () => EditProfile::getUrl()),
+                MenuItem::make()
+                    ->label('Profile Settings')
+                    ->url(fn () => EditProfile::getUrl())
+                    ->icon('heroicon-o-cog-6-tooth'),
```

Comment:
Based on the size of the icon I think it should be solid

```suggestion
                    ->icon('heroicon-s-cog-6-tooth'),
```